open_instruct/utils.py

```diff
@@ -385,6 +385,99 @@ def get_datasets(
     return raw_datasets
 
 
+def combine_dataset(
```

Comment:
@natolambert this is essentially the same as your `get_dataset` but focused on just mixing the splits of datasets without doing specialized logic.

I was going to use the `get_dataset` but it does not support other splits like `dev` or `validation`. We should refactor those logic and the handle different dataset logic out in the future.

Reply:
Yes we can refactor at future time. I had removed those features to reduce complexity :)